test/unit/orchestration/dhcp_test.rb

```diff
@@ -65,7 +65,8 @@ def teardown
   end
 
   test "provision interface DHCP records should contain filename/next-server attributes" do
-    ProxyAPI::TFTP.any_instance.expects(:bootServer).returns('192.168.1.1')
+    ProxyAPI::TFTP.any_instance.unstub(:bootServer)
```

Comment:
At this point, when is `bootServer` stubbed? Also why not leaving the `expects` so that it tests `bootServer` is called?